index.js

```diff
@@ -1,38 +1,71 @@
-function * entries (obj) {
-  for (let key of Object.keys(obj)) {
-    yield [obj[key], key]
-  }
-}
+'use strict'
```

Comment:
I really don't like the extra boilerplate, and it doesn't have any obvious benefits for this particular app. There's [this](https://github.com/isaacs/use-strict) but still not great.